### PR TITLE
Switched database from using config file to env vars

### LINKED ISSUE #76: Docker build is failing due to config.yaml not being present in the repository
We need to shift the logic from loading a configuration file, to exposing a series of environment variables in the scripts and updating the logic in timbot to use these env vars.

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ And appropriately set the following environment variables:
 - `$SLACK_CHANNEL_ID`: the channel ID number.
 - `$SLACK_TIMBOT_USER_ID`: the timbot slack user ID.
 
+To use the database, you'll have to set the following environment variables as well:
+
+- `$SQL_HOST`: Your MySQL host
```
Comment (timflannagan) [inline README.md]:
```suggestion
- `$SQL_HOST`: <Your MySQL host>
```

File: Jenkinsfile [hunk 2]
```diff
@@ -1,37 +0,0 @@
-pipeline {
```
Comment (Daniel-Pivonka) [inline Jenkinsfile]: Whats happening here

**Review by Daniel-Pivonka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nathan-weinberg) [reply README.md]: nah

File: Jenkinsfile (hunk 2 above)
Comment (nathan-weinberg) [reply Jenkinsfile]: nah

Comment (Daniel-Pivonka) [reply Jenkinsfile]: not sure why my comment was not address here

File: src/timbot.py [hunk 3]
```diff
@@ -166,20 +165,12 @@ def main():
...
-        print("Error loading configuration data: ", e)
-        sys.exit()
-
     # connect to database
     try:
-        conn = mysql.connector.connect(user=config['mysql']['user'],
-                                       password=config['mysql']['password'],
-                                       host=config['mysql']['host'],
-                                       database=config['mysql']['database'])
+        conn = mysql.connector.connect(user=os.environ['SQL_USER'],
```
Comment (timflannagan) [inline src/timbot.py:170]: This is going to produce a key error exception if that environment variable doesn't exist.

Comment (nathan-weinberg) [reply src/timbot.py:170]: better set it then

File: src/timbot.py [hunk 4]
```diff
@@ -166,20 +165,12 @@ def main():
...
+                                       password=os.environ['SQL_PASSWORD'],
```
Comment (Daniel-Pivonka) [inline src/timbot.py:171]: Key error needs to be handled here

**Review by Daniel-Pivonka — DISMISSED** [review]

File: src/timbot.py (hunk 4 above)
Comment (nathan-weinberg) [reply src/timbot.py:171]: see above comment

**Comment by timflannagan** [comment]
This is such a bad implementation.

File: README.md (hunk 1 above)
Comment (nathan-weinberg) [reply README.md]: this isn't how the other vars in the README are formatted

Comment (nathan-weinberg) [reply README.md]: @timflannagan1 @Daniel-Pivonka which direction do we want to go i, `<>` or no `<>`?

File: Jenkinsfile (hunk 2 above)
Comment (nathan-weinberg) [reply Jenkinsfile]: we are removing the file

File: src/timbot.py (hunk 3 above)
Comment (nathan-weinberg) [reply src/timbot.py:170]: even though this wasn't wrapped before, i will wrap it this time in next patchset

File: README.md (hunk 1 above)
Comment (timflannagan) [reply README.md]: `<>`

File: src/timbot.py [hunk 5]
```diff
@@ -166,20 +165,12 @@ def main():
...
+                                       host=os.environ['SQL_HOST'],
+                                       database=os.environ['SQL_DATABASE'])
```
Comment (timflannagan) [inline src/timbot.py:173]: We need to handle the case where these environment variables don't exist.

File: src/timbot.py [hunk 6]
```diff
@@ -166,22 +165,14 @@ def main():
...
 
-    except mysql.connector.Error as err:
+    except Exception as err:
```
Comment (timflannagan) [inline src/timbot.py:175]: lol

**Review by timflannagan — DISMISSED** [review]

File: README.md [hunk 7]
```diff
@@ -23,6 +23,13 @@ And appropriately set the following environment variables:
 - `$SLACK_CHANNEL_ID`: the channel ID number.
 - `$SLACK_TIMBOT_USER_ID`: the timbot slack user ID.
 
+To use the database, you'll have to set the following environment variables as well:
+
+- `$SQL_HOST`: <Your MySQL host>
+- `$SQL_USER`: <Your MySQL user>
+- `$SQL_PASSWORD`: <Your MySQL password>
+- `$SQL_DATABASE`: <Your MySQL database>
+
```
Comment (timflannagan) [inline README.md:32]: Really poor implementation

**Review by timflannagan — APPROVED** [review]

**Pull request merged**